unittests/Feature/FeatureModelWriter.cpp

```diff
@@ -0,0 +1,29 @@
+#include "vara/Feature/FeatureModelWriter.h"
+#include "vara/Feature/FeatureModelParser.h"
+
+#include "llvm/Support/MemoryBuffer.h"
+#include "gtest/gtest.h"
+
+#include <iostream>
+
+namespace vara::feature {
+
+const std::string TESTFILEDIRPATH = "../../../test/";
+
+TEST(XmlWriter, testtest) {
+  auto FS = llvm::MemoryBuffer::getFileAsStream(TESTFILEDIRPATH + "test.xml");
+  assert(FS && "Input file could not be read");
+  auto FM = FeatureModelXmlParser(FS.get()->getBuffer()).buildFeatureModel();
+
+  FeatureModelXmlWriter Fmxw = FeatureModelXmlWriter(*FM);
+  auto OutOpt = Fmxw.writeFeatureModel();
+  assert(OutOpt && "Feature model could not be written");
+
+  std::string Out = OutOpt.value();
+  FS = llvm::MemoryBuffer::getFileAsStream(TESTFILEDIRPATH + "test_out.xml");
```

Comment:
for the specific files, we should use better names that relate to the specific test case